file_formats/ie_formats/ini.htm

```diff
@@ -1,89 +1,408 @@
+        <br />
+
+        <strong>exit = name_of_group_section</strong><br />
+        This attribute expects one or more comma-separated names of group sections. <span class="unknown">It is likely that it's executed by the engine whenever the player leaves the area.</span> It is only supported by PST, IWD and IWD2.<br />
+        <br />
+
+        <strong>event = name_of_group_section</strong><br />
+        This attribute expects one or more comma-separated names of group sections. It is executed by the engine whenever the conditions defined by the assigned sections are met.<br />
+        <br />
       </div>
       <br />
 
 
-      <strong>[name_of_section1]</strong><br />
+      <strong>[name_of_group_section]</strong><br />
       <div class="indent1">
-        Section starts by its identifier<br />
-        <strong>critters= cre1,cre2,...</strong><br />
-        This assigns spawned creatures identifiers to the [name_of_section1] section. Creatures identifiers will be used later as the names of appropriate section names.<br />
-    
-        <strong>interval= integer number</strong><br />
-        Time interval before creatures from [name_of_section1] will be spawned. Time is counted in relation 1 minute (of real or game time ?) = 900.<br />
-        The similar definition is given in case of [name_of_section2] or other sections defined by user in [spawn_main] section.<br />
-        At last spawned creatures identifier definitions.<br />
+        This section controls a group of individual spawn sections. It supports the following attributes:<br />
+        <br />
+
+        <strong>critters = name_of_creature_section1,name_of_creature_section2,...</strong><br />
+        This attribute expects a comma-separated sequence of creature section names. These sections define individual creatures with their spawn conditions, attributes and behavior. Each listed creature section is evaluated and executed when their spawn conditions are met.<br />
+        <br />
+
+        <strong>interval = number</strong><br />
+        Time interval, measured in AI ticks (15 ticks = 1 second), before creature sections defined by the "critters" attribute are evaluated. This attribute is only relevant if the section is linked to the "event" attribute of the [spawn_main] section.<br />
+        <br />
+
+        <strong>detail_level = name</strong><br />
+        <span class="unknown">The meaning of this attribute is unknown, possibly a memory saving option.</span> Known attribute values: <span class="code">High</span>. It is only supported by PST.<br />
+        <br />
+
+        <strong>control_var = ???</strong><br />
+        <span class="unknown">The meaning of this attribute is unknown.</span> <strong>Note:</strong> In BGEE specifying this attribute causes the game to crash.<br />
+        <br />
+
+        <strong>spawn_time_of_day = ???</strong><br />
+        <span class="unknown">The meaning of this attribute is unknown, possibly unused.</span><br />
```

Comment:
```suggestion
        <span class="unknown">The meaning of this attribute is unknown</span>, and it is unused in PST and IWD2.<br />
```
I couldn't check IWD, since I only have a non-tolowered install and weidu bailed out. Also, when Avenger wrote that it's not working, I don't know which engine he referred to — all three have the key in the exe.